src/test/java/SpaceIDConverterTest.java

```diff
@@ -0,0 +1,19 @@
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class SpaceIDConverterTest {
+    @Test
+    public void itConvertsAnIntegerToASpace() {
+        SpaceIDConverter converter = new SpaceIDConverter(3, 3);
+        assertEquals(new Space(0, 0), converter.convert(1));
+        assertEquals(new Space(1, 0), converter.convert(2));
+        assertEquals(new Space(2, 0), converter.convert(3));
+        assertEquals(new Space(0, 1), converter.convert(4));
+        assertEquals(new Space(1, 1), converter.convert(5));
+        assertEquals(new Space(2, 1), converter.convert(6));
+        assertEquals(new Space(0, 2), converter.convert(7));
+        assertEquals(new Space(1, 2), converter.convert(8));
+        assertEquals(new Space(2, 2), converter.convert(9));
+    }
```

Comment:
Do you think I need to handle unexpected input here? I am on the fence.

One the one hand, this will probably only ever be used by the `UI` class, which validates and parses the input before it sends it to the Converter.

At the same time, if it did somehow get sent a value like `-1` it will send back a `Space` that doesn't exists on the board.

I considered throwing an exception or maybe making an assertion. Thoughts?